### PR TITLE
fix: nested GitLab query batching

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix https://github.com/ParabolInc/parabol/issues/6540

Previously, we would batch the `getIssue` query that was sent to GitLab for all 50 issues. 50 queries * a complexity score of ~15 exceeded the maximum score allowed by GitLab's API of 250. 

We're now setting a `maxBatchSize` of 5, so only 5 queries will be batched together. 

Loom demo going through repro steps: https://www.loom.com/share/2b86664e9a9b4ca3bafaceac9c59577a

### CODE REVIEW

File: packages/client/types/constEnums.ts [hunk 1]
```diff
@@ -369,7 +369,7 @@ export const enum Threshold {
   MAX_POKER_DIMENSION_NAME = 50,
   MAX_REACTJIS = 12,
   MAX_POKER_STORIES = 50,
-  MAX_GITLAB_POKER_STORIES = 15,
+  MAX_GITLAB_POKER_STORIES = 50, // TODO: remove this and use MAX_POKER_STORIES instead
```
Comment (nickoferrall) [inline packages/client/types/constEnums.ts:372]: I'll remove this and refactor in a separate PR to keep this PR as small as possible

File: packages/server/integrations/gitlab/GitLabServerManager.ts [hunk 2]
```diff
@@ -49,7 +49,8 @@ class GitLabServerManager implements TaskIntegrationManager {
         info,
         endpointContext: {
           accessToken: this.auth.accessToken,
-          baseUri: this.serverBaseUrl
+          baseUri: this.serverBaseUrl,
+          dataLoaderOptions: {maxBatchSize: 5}
```
Comment (nickoferrall) [inline packages/server/integrations/gitlab/GitLabServerManager.ts:53]: Previously, I wasn't including the `maxBatchSize` in the `GitLabServerManager`

**Comment by nickoferrall** [comment]
Hey @mattkrick, here's the fix with the `maxBatchSize`. Are you happy for me to merge your nest-gql PR https://github.com/ParabolInc/nest-graphql-endpoint/pull/10 and then use the regular package in this PR rather than the branch?

**Comment by nickoferrall** [comment]
Hey @igorlesnenko, could you review this PR, please? Once this is merged we should be able to release GitLab to our users

**Review by igorlesnenko — APPROVED** [review]

**Comment by Dschoordsch** [comment]
It's 3rd in the queue atm, so should work out fine.

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**